automatic/poi/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,19 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+  packageName   = $env:ChocolateyPackageName
+  fileType      = 'exe'
+  file          = "$toolsDir\"
+
+  softwareName  = 'poi*'
+
+  silentArgs    = '/S'
```

Comment:
I see that this software with this is only installing as the current user, is it possible to pass in an argument to have it install as All Users instead.
If possible, all users installation should always be used with chocolatey, due to the nature of the software.

Reply:
Fixed in fbae7a1